osbrain/tests/test_proxy.py

```diff
@@ -323,4 +323,4 @@ def shoot(self):
     assert time.time() - t0 < 0.2
     assert not wayne._next_oneway
 
-    assert wait_agent_attr(wayne, value=20*['bang!'], timeout=1.2)
+    assert wait_agent_attr(wayne, value=20*['bang!'], timeout=1.5)
```

Comment:
Was this failing with the previous timeout?

Reply:
Maybe at some point, I wouldn't have changed it otherwise. I'll run it through appveyor again, maybe it needed longer timeouts before, since we were using 8 threads instead of 2.

Reply:
Is still fails if set back to `1.2`?

Reply:
Same as above, I ran it in a loop to see if it would eventually fail, and after 100-200 tests I've gotten it to fail a couple of times. 1.5 looks like a safer value.